6.x/crud-filters.md

```diff
@@ -5,63 +5,176 @@
+
+<a name="fluent-filters-other-chained-methods"></a>
+#### Other Chained Methods
+
+If you chain the following methods to a ```CRUD::filter('name')```, they will do something very specific instead of adding that attribute to the filter:
+
+- ```->remove()``` - By chaining **remove()** on a filter you remove it from the current operation;
+
+```php
+CRUD::filter('name')->remove();
+```
+
+- ```->forget('attribute_name')``` - By chaining **forget('attribute_name')** to a filter you remove that attribute from the filter;
+
+```php
+CRUD::filter('name')->forget('suffix');
 ```
-> Notes about the filter logic closure
-> - the code will only be run on the controller's ```index()``` or ```search()``` methods;
-> - you can get the filter value by specifying a parameter to the function (ex: ```$value```);
-> - you have access to other request variables using ```$this->crud->getRequest()```;
-> - you also have read/write access to public properties using ```$this->crud```;
-> - when building complicated "OR" logic, make sure the first "where" in your closure is a "where" and only the subsequent are "orWhere"; Laravel 5.3+ no longer converts the first "orWhere" into a "where";
+
+- ```->after('destination')``` - By chaining **after('destination_filter_name')** you will move the current filter after the given filter;
+
+```php
+CRUD::filter('name')->after('description');
+```
+
+- ```->before('destination')``` - By chaining **before('destination_filter_name')** you will move the current filter before the given filter;
+
+```php
+CRUD::filter('name')->before('description');
+```
+
+- ```->makeFirst()``` - By chaining **makeFirst()** you will make the current filter the first one for the current operation;
+
+```php
+CRUD::filter('name')->makeFirst();
+```
+
+- ```->makeLast()``` - By chaining **makeLast()** you will make the current filter the last one for the current operation;
+
+```php
+CRUD::filter('name')->makeLast();
+```
+
+<a name="filter-logic-closures"></a>
+#### Filter logic closures
+
+Backpack filters do not contain any default filtering _logic_, because it cannot infer that. If you use a filter and don't specify a filter logic, no filtering of entries will actually happen. You have to define that, inside a _closure_.
+
+Filter logic closures are just an anonymous functions that gets executed when the filter is active. You can use any Laravel or Backpack functionality you want inside them. For example, a valid closure would be:
+
+```php
+(function($value) {
+  CRUD::addClause('where', 'draft', 1);
+})
+```
```

Comment:
```php
CRUD::filter('draft')
 ->whenActive(function($value) {
  CRUD::addClause('where', 'draft', 1);
 })
```
I would prefer to have a complete code block. Easier to copy and paste for the user.

Reply:
Sounds good. Heads-up, you can use the Github suggest button here, to make a suggestion. Then it'll be easy for the reviewer to add your code to the codebase:

![CleanShot 2023-08-11 at 15 19 14@2x](https://github.com/Laravel-Backpack/docs/assets/1032474/c4182619-bc6a-4311-8548-87103912c313)

Reply:
> Sounds good. Heads-up, you can use the Github suggest button here, to make a suggestion. Then it'll be easy for the reviewer to add your code to the codebase:
> 
> ![CleanShot 2023-08-11 at 15 19 14@2x](https://user-images.githubusercontent.com/1032474/260004389-c4182619-bc6a-4311-8548-87103912c313.png)

Too late:
![image](https://github.com/Laravel-Backpack/docs/assets/7188159/b2d1fd31-50c1-43ee-8390-f39072d239be)

Reply:
Haha perfect. Closing then.